apps/ios/SimpleXChat/APITypes.swift

```diff
@@ -530,99 +530,99 @@ public enum ChatResponse: Decodable, Error {
             case .chatRunning: return noDetails
             case .chatStopped: return noDetails
             case .chatSuspended: return noDetails
-            case let .apiChats(chats): return String(describing: chats)
-            case let .apiChat(chat): return String(describing: chat)
-            case let .userSMPServers(smpServers, presetServers): return "smpServers: \(String(describing: smpServers))\npresetServers: \(String(describing: presetServers))"
-            case let .smpTestResult(smpTestFailure): return String(describing: smpTestFailure)
-            case let .chatItemTTL(chatItemTTL): return String(describing: chatItemTTL)
+            case let .apiChats(_, chats): return String(describing: chats)
```

Comment:
the point of these strings is to debug... it's probably better to include userId as the first line in console response views, so we can see what it is...

I would probably define something like `showUser(user, <current string>)` that prepends it, it'll be less noisy than if you just change interpolations everywhere...

Reply:
I named it `withUser`

Reply:
https://github.com/simplex-chat/simplex-chat/pull/1793/commits/1e80b354db519fa470f40c7a4d8bf072d13efca4